eng/Generate.ps1

```diff
@@ -119,7 +120,7 @@ if (!($Exclude -contains "Samples"))
         $swaggerDefinitions[$projectName] = @{
             'projectName'=$projectName;
             'output'=$projectDirectory;
-            'arguments'="--require=$sampleConfigurationPath"
+            'arguments'="--require=$sampleConfigurationPath --launch-debugger"
```

Comment:
This was me trying to get the debugger working, but I ended up not using it this way.